relay/src/main.rs

```diff
@@ -27,7 +27,14 @@ use methods::GUEST_LIST;
 use risc0_zkvm::sha::Digest;
 
 /// Index 0 private key generated by default in Anvil.
-const ANVIL_DEFAULT_KEY: &str = "ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80";
+const ANVIL_DEFAULT_KEY: &'static str =
```

Comment:
This should be a clippy error but: you shouldn't need `&'static` for const &str's because its implied